examples/motoko_examples/factorial/src/factorial.ts

```diff
@@ -0,0 +1,18 @@
+import {
+    Query,
+    nat
+} from 'azle';
+
+// Calculate the product of all positive integers less than or equal to `n`.
+export function fac(n: nat): Query<nat> {
+    return go(n);
+}
+
+// We implement the recursion in a helper function.
+function go(m: nat): nat {
+    if(m == 0n) {
```

Comment:
We should of course have linters and prettifiers do this for us, but the if statement should have a space after the if